### ISSUE TITLE
Can't import 'watch later' list into grayjay

### ISSUE THREAD

**Issue by AliensInc** [futo-org/grayjay-android#1623]
### What happened?

Device: Pixel 7A
OS: Lineage OS 21 (Android 14)
App: YT-addon

I usually create a pretty big 'watch later' list in YT than later import it and watch it in grayjay.
Since just over a week the import gets an error.
This happens just as I push
'Import playlist' inside the YT source.

### Grayjay Version

276

### What plugins are you seeing the problem on?

Youtube

### Plugin Version

219

### When do you experience the issue?

- [X] While logged in
- [ ] While logged out
- [ ] N/A

### Relevant log output

```shell
Failed to retrieve playlists.
[youtube] Cannot read properties of null
(reading 'contents') (1671)[28-29]

source.getUserplaylist(...)
```

**Comment by AliensInc** [comment]
This also happened with the previous version of grayjay

**Comment by Zvonimir-FUTO** [comment]
How big of a list are we talking? I will try to reproduce it.

**Comment by AliensInc** [comment]
Right now 148 vids.
It has worked with over 250 before that.

My guess is that G has changed something as this worked and also stopped working within the same versions for both grayjay and the YT-addon.

**Comment by PublioPapinioStazio** [comment]
why? have you ever managed to import the 'watch later' playlist from youtube?
it has NEVER worked for me, since the first time I installed grayjay a year ago
it simply tells me that there is nothing to import (when in reality I have about thirty videos in my playlist)
and now for a few weeks it hasn't even let me import the subscriptions...

**Comment by AliensInc** [comment]
Yep, did that for months as it's without adverts but after some update of something, guess google, it stopped working giving me an error

**Comment by AliensInc** [comment]
With the update of the YT-addon I got today 250221(ISO 8601) work.
Version 238.

The import of 'watch later' now works.

thx :D
